eng/build.yml

```diff
@@ -133,19 +130,29 @@ jobs:
             Token: $(dn-bot-dnceng-artifact-feeds-rw)
 
     - script: >-
-        $(Build.SourcesDirectory)/build$(scriptExt)
-        -ci
+        $(Build.SourcesDirectory)/eng/cibuild$(scriptExt)
         -test
         -configuration ${{ parameters.configuration }}
-        -prepareMachine
+        -architecture ${{ parameters.architecture }}
         $(_InternalInstallArgs)
         $(_InternalBuildArgs)
-        $(_InternalSignArgs)
-        $(_InternalPublishArgs)
       displayName: Build and Test
       condition: succeeded()
-
-    # # Publish test results to Azure Pipelines
+    
+    # Native build does not support signing. Move pack, sign, and publish
```

Comment:
We still do authenticode on windows for native dll's right?

Reply:
It should. We aren't building any dlls yet though, so I cannot verify that at this time.